### PR TITLE
use redis session as default one for docker

### PR DESCRIPTION
**Pull request by inkhey**
[skip ci]

*Please indicate here a summary of the pull request content. If a related issue exists, please indicate issue reference here*

## Checkpoints

The goal of this section is to help code reviewers and the merge request author to do the required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewers**

- [ ] The code is clear enough
- [ ] If there are FIXMEs in the code, then there are associated issues and issue is referenced in the FIXME
- [ ] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present
- [ ] Automated tests have been written (covering the feature or non regression), or *at least* an issue has been created to implement the test

**For developers**

- [ ] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [ ] The original issue has been updated with a short summary of the implemented solution

**For the quality team**

- [ ] Tested by the quality team

### CODE REVIEW

File: tools_docker/Debian_Uwsgi/common.sh [hunk 1]
```diff
@@ -16,6 +16,8 @@ if [ ! -f /etc/tracim/development.ini ]; then
     sed -i "s|^basic_setup.preview_cache_dir = .*|basic_setup.preview_cache_dir = /var/tracim/data/preview|g" /etc/tracim/development.ini
     sed -i "s|^basic_setup.sessions_data_root_dir = .*|basic_setup.sessions_data_root_dir = /var/tracim/data|g" /etc/tracim/development.ini
     sed -i "s|^basic_setup.api_key =.*|basic_setup.api_key = $KEY|g" /etc/tracim/development.ini
+    sed -i "s|^session.type =.*|session.type = ext:redis|g" /etc/tracim/development.ini
```
Comment (PhilippeAccorsi) [inline tools_docker/Debian_Uwsgi/common.sh:19]:
```suggestion
    sed -i "s|^session.type = .*|session.type = ext:redis|g" /etc/tracim/development.ini
```
Without space, regex not working.

File: tools_docker/Debian_Uwsgi/common.sh [hunk 2]
```diff
@@ -16,6 +16,8 @@ if [ ! -f /etc/tracim/development.ini ]; then
...
+    sed -i "s|^session.url =.*|session.url = redis://localhost:6379/0|g" /etc/tracim/development.ini
```
Comment (PhilippeAccorsi) [inline tools_docker/Debian_Uwsgi/common.sh:20]: Make sure `session.url =` have no space after `=` in development.ini  If its ok, its good :)

File: tools_docker/Debian_Uwsgi/common.sh (hunk 1 above)
Comment (inkhey) [reply tools_docker/Debian_Uwsgi/common.sh:19]: in fact, regex does work without spacing (.* will match space too), but you're right it's better this way.

File: tools_docker/Debian_Uwsgi/common.sh (hunk 2 above)
Comment (inkhey) [reply tools_docker/Debian_Uwsgi/common.sh:20]: checked.

**Pull request merged**